src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -473,6 +508,7 @@ public function render()
 
 		$cssClasses = implode(' ', $this->breadcrumbsCssClasses);
 		return '<'. $this->listElement .' class="' . $cssClasses .'">' 
+		       . $this->icon
```

Comment:
I think this is where we should do the actual "rendering", depending on whether or not `$this->icon` is `null`.